### DISCUSSION TITLE
Comparing Modflow input packages model

### DISCUSSION THREAD

**Discussion by Hperea** [modflowpy/flopy#1281] [Q&A]
Is it possible to use  flopy to compare MF input packages from two models? 
Thanks in advance,

**Comment by christianlangevin** [comment]
That would be a nice feature and perhaps something to think about, though I'm not sure how much it would get used.  If we did decide to pursue, we might think about implementing operators for packages, models, and the underlying data types.

**Reply by jlarsen-usgs** [reply]
It might be worthwhile to start with adding "operator" functionality to some of the underlying data types to allow for easy comparisons of arrays, performing sliced operations in place, etc... What I'm imagining from the user side is something like:

ml.npf.k[0, :, 1:3] *= 4

instead of calling the array and then resetting it.
